documentation-website/Writerside/topics/DSL-Table-Types.topic

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+        SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       title="Table types" id="DSL-Table-Types">
+    <show-structure for="chapter,procedure" depth="2"/>
+    <p>
+        The DSL API of Exposed provides two classes to define tables:
+    </p>
+    <list>
+        <li>The core <a anchor="table"><code>Table</code></a> class.</li>
+        <li>
+            The <a anchor="IntIdTable"><code>IntIdTable</code></a> class for simplified <code>id</code> columns.
+        </li>
```

Comment:
I'm not sure if there should be a dedicated section (above and below) in the DSL topic for `IntIdTable`. Or if it stays, it should probably be generalized to `IdTable` instead of singling out only 1 of the subclasses.

We've had some users in the past comment about why DSL samples are using `IdTable` types, which rely on `EntityID`, despite the latter being heavily associated with the DAO approach. And also with the DSL docs making no mention about this wrapping class and how, for example, `id` column values may need to be unwrapped when accessed like `StarWarsFilms.id.value`.

Imo, these `IdTable` types are located in the core module, so they don't require a dependency on the dao module. So if they are immediately available to DSL-only users, they should be described in some capacity.
But maybe we should emphasize `Table` as the core class and then mention `IdTable` just as a convenience class that extends `Table` and defines the primary key etc for you.
I wonder if there should also be a link to the DAO page in case a DSL user wants to know why their `id` column is of type `Column<EntityID<Int>>` and what that means...

@e5l What do you think?

Reply:
@vnikolova Took a look at the new topic and it looks great. Very clear now.